### DISCUSSION TITLE
Search improvement

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#566] [Q&A] [answered]
The long press on Search button is not used so far. Could you please make a new function for search without POI search?

The POI search is very time consuming. It takes too long for address seach since the POIs are often big and have search priority. When deactivating POI seach it is much quicker. Could you then implement this new function with long press to search Online with priority, routes and tracks with 2nd priority, no POI search?

**Comment by devemux86** [comment]
@DolomitesTom thanks for the suggestion.

We can certainly think about this.
Search history is another improvement I have in the list.

**Comment by devemux86** [comment]
> The long press on Search button is not used so far. Could you please make a new function for search without POI search?

This will be available in the next app update.

**Comment by devemux86** [comment, marked as answer]
- Search with long press without offline POI is available in https://github.com/devemux86/cruiser/discussions/641
